test/system/liquidation.test.js

```diff
@@ -166,7 +166,7 @@ describeFn("System -- liquidation happy path", () => {
     })
 
     it("should consume a reasonable amount of gas", async () => {
-      await expect(parseInt(tx.gasLimit)).to.be.lessThan(480000)
+      await expect(parseInt(tx.gasLimit)).to.be.lessThan(485000)
```

Comment:
Not in this PR but we quickly need to figure out why such high gas limit is needed. Estimations in earlier PRs about purchasing signer bonds were saying about ~250k.

Reply:
Yep, I will try to figure out this separately.